cognite_toolkit/_cdf_tk/prototypes/resource_loaders.py

```diff
@@ -17,18 +17,197 @@
     ThreeDModelUpdate,
     ThreeDModelWrite,
     ThreeDModelWriteList,
+    TimeSeries,
+    TimeSeriesList,
+    TimeSeriesWrite,
+    TimeSeriesWriteList,
     capabilities,
 )
 from cognite.client.data_classes.capabilities import Capability
 from cognite.client.exceptions import CogniteAPIError, CogniteNotFoundError
 from cognite.client.utils.useful_types import SequenceNotStr
 
 from cognite_toolkit._cdf_tk._parameters import ParameterSpec, ParameterSpecSet
-from cognite_toolkit._cdf_tk.loaders._base_loaders import ResourceContainerLoader, ResourceLoader
-from cognite_toolkit._cdf_tk.loaders._resource_loaders import DataSetsLoader, LabelLoader
+from cognite_toolkit._cdf_tk.loaders._base_loaders import (
+    ResourceContainerLoader,
+    ResourceLoader,
+)
+from cognite_toolkit._cdf_tk.loaders._resource_loaders import (
+    DataSetsLoader,
+    LabelLoader,
+)
 from cognite_toolkit._cdf_tk.utils import CDFToolConfig, load_yaml_inject_variables
 
 
+@final
+class TimeSeriesLoader(ResourceLoader[str, TimeSeriesWrite, TimeSeries, TimeSeriesWriteList, TimeSeriesList]):
```

Comment:
What is the motivation for implementing this one? We already have a TimeSeriesLoader in the core code base? 

Is it only to create a new load method? And allow parquet files and csv files? Then, we should dynamically only modify these methods on runtime when the flag is set. You can see how the file and timeseries loaders are modified in the `setup_asset_loader`. 

However, I am leaning towards waiting with this functionality and instead only implement the dump command and not modify the loader.

Reply:
> I suggest, removing the `TimeSeriesLoader` you added and save support for loading parquet and csv until `0.3.0a1` (as it is a bit messy to implement runtime modification of classes as we do for prototypes).

> What is the motivation for implementing this one? 

It was a copy/paste approach mimic `dump_assets.py` / `AssetLoader` as template
```
from cognite_toolkit._cdf_tk.prototypes.resource_loaders import AssetLoader
```

Only the `folder_name` attribute is used, but I didn't want to mess up the symmetry (w/o knowing about all details).
What I can do instead is simply define `time_series_folder_name` locally in `dump_timeseries.py` and remove the `TimeSeriesLoader`

that's fine?